pom.xml

```diff
@@ -217,6 +216,49 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <version>0.12</version>
+                <configuration>
+                    <useDefaultExcludes>true</useDefaultExcludes>
+                    <useMavenDefaultExcludes>true</useMavenDefaultExcludes>
+                    <useIdeaDefaultExcludes>true</useIdeaDefaultExcludes>
+                    <useEclipseDefaultExcludes>true</useEclipseDefaultExcludes>
+                    <excludeSubProjects>true</excludeSubProjects>
+                    <excludes>
```

Comment:
I think we may need some additional excludes. The first that came to mind are the OMAG Server configuration files, which currently end up being put in the root directory.

There are also other files created during a build such as test results.

The top level .gitignore is a start - could we add that as an excludesFile (not tried it?) - even though some are already explicitly listed and/or covered by the IDEA exclusions etc

Reply:
There are 62 implicit excludes. Among them .gitignore. 
The OMAGApplication configuration creates a .json file which is excluded.
Of course I have tested it, build the project with it and works.

Reply:
The OMAG server config file is called *.config rather than *.json
The cohort registry file is called *.registrystore
The compliance test results file is called *.results